File: solution/PeopleRegistrationApi/Models/Person.cs
public class Person : Base
{
    public int Id { get; set; }
    
    [Required]
    public string Name { get; set; }
    
    public PersonGender Gender { get; set; }
    
    [EmailAddress]
    public string Email { get; set; }
    
    [Required]
    [DataType(DataType.Date)]
    public DateTime DateOfBirth { get; set; }
    
    public string Nationality { get; set; }
    
    [Required]
    [RegularExpression(@"^\d{3}\.\d{3}\.\d{3}-\d{2}$", ErrorMessage = "Invalid CPF format.")]
    public string Identification { get; set; }
}

File: src/types/Person.ts
export interface Person {
    id?: number;
    name: string;
    gender?: string;
    email?: string;
    dateOfBirth: string; // ISO format date
    nationality?: string;
    identification: string; // Must be validated for format and uniqueness
    createdAt?: Date;
    updatedAt?: Date;
}